.github/workflows/cd-syft-dev.yml

```diff
@@ -0,0 +1,167 @@
+name: CD - Syft - Deploy `dev` to K8s
+
+on:
+  # TODO: Re-enable once merged & tested
+  # schedule:
+  #   - cron: "0 */3 * * *"
+
+  workflow_dispatch:
+    inputs:
+      none:
+        description: "Run Tests Manually"
+        required: false
+
+jobs:
+  deploy-syft-dev:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Checkout Openmined/infrastructure
+        uses: actions/checkout@v4
+        with:
+          repository: OpenMined/infrastructure
+          path: infrastructure
+
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.11"
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
```

Comment:
Since we're building for `linux/amd64` I don't think we need this. @rasswanth-s can you please confirm?

Reply:
Yep , we could remove the same 👍